src/main/java/uk/gov/companieshouse/pscstatement/delta/serialization/ChsDeltaSerializer.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.companieshouse.pscstatement.delta.serialization;
```

Comment:
Optional change - This class is currently only used by a unit test. While we will eventually need a serializer for retryables I'm not sure we should be adding it in purely to assist in a unit test for another class.

If we are going to keep it in could a unit test be written up for it?